mitodl/mitxonline
# Adding course topics

### What are the relevant tickets?
Fix https://github.com/mitodl/hq/issues/4110
### Description (What does it do?)
Adding topics in MITx Online. 
The ability to create delete topics through the cms, and add them to courses.

Populates topics with existing department names. I don't think it is necessary, but this is how it was done in mitxpro. And I think it is nice to have some topics to begin with.

### Screenshots (if appropriate):
<img width="790" alt="Screenshot 2024-05-16 at 8 53 26 AM" src="https://github.com/mitodl/mitxonline/assets/7574259/cd1c7f03-2d77-4800-9f24-d0d3653930a6">


### How can this be tested?

Run migrations
Run the app, go to the cms and make sure you can see Course Topics.
Click on it and try to add or delete a course topic.

Go to Course page add topic to a course page.

Also view `http://mitxonline.odl.local:8013/api/v2/parent_course_topics/`
